lib/unicodeMappings.json

```diff
@@ -32,6 +33,7 @@
   "®": "੍ਰ",
   "´": "ੵ",
   "µ": "ੰ",
+  "μ": "ੰ",
```

Comment:
Is this not duplication?

Reply:
If you inspect both chars, they are different (U+00B5 and U+03BC respectively) but are still assigned in the ascii font.

Reply:
Ah, thought something like this might be the case 👍